docs/guides/release-notes.md

```diff
@@ -12,15 +12,25 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
 
 ---
 
-``Last updated: R3.1.4 (UserConnect-7.1.3006)``
+``Last updated: R3.1.5 (UserConnect-7.1.4105)``
 
 ---
 
-## 3.1.5 (TBD, 2024)
-###
-- Supported Controller version: **UserConnect-TBD**
+## 3.1.5
+### Notes:
+- Release date: **(18 July 2024)**
+- Supported Controller version: **UserConnect-7.1.4105**
 - Supported Terraform version: **v1.x**
 
+### Bug Fixes:
+1. Fixed issue in Firenet and FQDN Gateway for Azure where we no longer require special handling of ``fqdn_lan_interface``  and ``lan_interface``.
```

Comment:
(comment for CHANGELOG applies for release notes)

Reply:
Can you say more, I don't understand our comment.

Reply:
Nevermind, I misunderstood